wisdem/glue_code/gc_LoadInputs.py

```diff
@@ -1504,10 +1504,6 @@ def write_ontology(self, wt_opt, fname_output):
                 self.wt_init["costs"]["bearing_mass_cost_coeff"] = float(
                     wt_opt["tcc.main_bearing_cost"] / wt_opt["tcc.main_bearing_mass"]
                 )
```

Comment:
I don't think this can just be deleted.  I think we need a similar if-block with the new cost structure.  The issue is if the user bypasses the torque-cost scaling with a user-input or the old method, then the output yaml needs to adjust the torque-cost scaling coefficient.  Otherwise, if that output yaml is then used as an input, the cost might not be consistent.  Hope that makes sense. . .